Train_pnr_status/script.py

```diff
@@ -0,0 +1,100 @@
+from selenium import webdriver
+from PIL import Image
+import pytesseract
+import io
+import numpy as np
+import cv2 as cv
+import time
+import tkinter   
+from tkinter import ttk
+
+
+def click():
+    global inputpnrnumber
+    inputpnrnumber = str(name.get()) 
+    if (inputpnrnumber.isnumeric()) and ( len(inputpnrnumber) == 10 ) :
+        window.destroy()
+
+
+def GUI() :
+    window.title("PNR STATUS")
+    lbl = ttk.Label(window, text = "                         ")
+    lbl.grid(column = 0, row = 0) 
+    lbl = ttk.Label(window, text = "    Enter PNR Number:    ")
+    lbl.grid(column = 0, row = 1)
+    nameEntered = ttk.Entry(window, width = 12, textvariable = name)
+    nameEntered.grid(column = 1, row = 1)
+    lbl = ttk.Label(window, text = "            ")
+    lbl.grid(column = 2, row = 1)
+    lbl = ttk.Label(window, text = "                        ")
+    lbl.grid(column = 1, row = 2) 
+    button = ttk.Button(window, text = "Submit", command = click)
+    button.grid(column = 1, row = 3)
+    lbl = ttk.Label(window, text = "                         ")
+    lbl.grid(column = 1, row = 4) 
+    window.mainloop()
+
+GUI()
+window = tkinter.Tk()
+name = tkinter.StringVar()    
+url="http://www.indianrail.gov.in/enquiry/PNR/PnrEnquiry.html?locale=en"
+
+google = input(r"Enter google executable path");
```

Comment:
which google executable? I am not aware of any google.exe. Can you clarify this?

Reply:
Sir this is to open the chrome, the path for chrome should be entered

Reply:
oh then you should specify Google Chrome and not Chrome, to avoid confusions